azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeCallStartTimeEvent.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import java.util.Date;
+
+/**
+ * Event with call start time.
+ */
+public class CallCompositeCallStartTimeEvent {
+    private final Date startTime;
+
+    /**
+     * Create {@link CallCompositeCallStartTimeEvent} with call start time.
+     *
+     * @param startTime call start time.
+     */
+    CallCompositeCallStartTimeEvent(final Date startTime) {
```

Comment:
Do we anticipate more properties in the Event? ARB will ask why not to use `Date` : `CallCompositeEventHandler<Date>`